src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -177,7 +184,6 @@ public final StreamConfiguration apply(final String scopeName) {
 			final StreamConfiguration streamConfig = StreamConfiguration
 				.builder()
 				.scalingPolicy(scalingPolicy)
-				.scope(scopeName)
```

Comment:
Am I right it was an initial scope name and, removing this, we're leaving the name generation to Pravega?

Reply:
it was a deprecated method in 0.6 and it's completely removed in 0.7. Stream creation requires scope name, stream name and stream config. So, this parameter is excessive now.